pkg/codegen/templates/constants.tmpl

```diff
@@ -0,0 +1,7 @@
+{{- if gt (len .SecuritySchemeProviderNames) 0 }}
+const (
+{{range $ProviderName := .SecuritySchemeProviderNames}}
+    {{- $ProviderName | ucFirst}}Scopes = "{{$ProviderName}}.Scopes"
```

Comment:
This is useful when generating a server, less so with a client
I didn't want to overcomplicate the PR by generating this only when `server|chi-server` is chosen, but maybe I should?